openshift/kustomize/shared_resources/base/pvc.yaml

```diff
@@ -1,45 +1,45 @@
----
-kind: PersistentVolumeClaim
-apiVersion: v1
-metadata:
-  name: api-storage
-  namespace: default
-  labels:
-    name: api
-    part-of: tno
-    version: 1.0.0
-    component: api
-    managed-by: kustomize
-    created-by: jeremy.foster
-spec:
-  # Storage class name is the type of storage [netapp-file-standard, netapp-file-extended, netapp-file-backup, netapp-block-standard, netapp-block-extended]
-  storageClassName: netapp-file-backup
-  # Storage access mode [ReadWriteOnce, ReadWriteMany]
-  accessModes:
-    - ReadWriteMany
-  resources:
-    requests:
-      storage: 10Gi
----
-# Storage for videos.
-kind: PersistentVolumeClaim
-apiVersion: v1
-metadata:
-  name: ingest-storage
-  namespace: default
-  annotations:
-    description: Storage for ingested audio and video files.
-    created-by: jeremy.foster
-  labels:
-    name: ingest-storage
-    part-of: tno
-    version: 1.0.0
-    component: ingest-storage
-    managed-by: kustomize
-spec:
-  storageClassName: netapp-file-backup
-  accessModes:
-    - ReadWriteMany
-  resources:
-    requests:
-      storage: 5Gi
+# ---
```

Comment:
Is this removed for a reason?  Don't we need this for the various Ingest Services?